web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -23,9 +30,11 @@
   import { handlePromiseError } from '$lib/utils';
   import { selectAllAssets } from '$lib/utils/asset-utils';
   import { navigate } from '$lib/utils/navigation';
+  import { page } from '$app/stores';
 
   export let isSelectionMode = false;
   export let singleSelect = false;
+  export let partOfRoute: boolean;
```

Comment:
I find the name a bit hard to understand right away. Is there a better name for this?